src/connectRoutes.js

```diff
@@ -115,8 +115,8 @@ export default (
   }
 
   const {
-    location: locationKey = 'location',
-    title: titleKey = 'title',
+    selectLocationState = state => state.location,
```

Comment:
Other than that, there's also the Flow errors which you can see in Travis. You can run `npm run flow` to see em from the console. You need to change the signature for `location` + `title` in `src/flowTypes.js`. Obviously now it's signature supports both a string and a function.